include/chemfiles/Format.hpp

```diff
@@ -28,7 +34,7 @@ class CHFL_EXPORT Format {
     Format(Format&&) = delete;
     Format& operator=(Format&&) = delete;
 
-    /// @brief Read a specific step from the trajectory file.
+    /// Read a specific `step` from the trajectory file.
```

Comment:
This is extremely nit-picky, but I prefer *italics* for concepts that are not code. Since there is no method called `step` I would use italics instead of backticks - but that's extremely nit-picky.

Reply:
The idea is more to use backticks for parameter to the function, describing them directly in the main description.

Reply:
Ok, that's fine!